.github/workflows/dis-vault-release.yml

```
@@ -0,0 +1,83 @@
+    if: startsWith(github.ref, 'refs/tags/dis-vault-v')
+    environment: flux-release
+    defaults:
+      run:
+        working-directory: ./services/dis-vault-operator/config
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@8e8c483db84b4bee98b60c0593521ed34d9990e8 # v6.0.1
+      - name: Setup vars
+        id: vars
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2086:info:2:22: Double quote to prevent globbing and word splitting

(shellcheck)